deploy-service/common/src/main/resources/sql/deploy.sql

```diff
@@ -171,6 +171,7 @@ CREATE TABLE IF NOT EXISTS agents (
     first_deploy   TINYINT(1)          NOT NULL DEFAULT 0,
     first_deploy_time     BIGINT,
     stage_start_date     BIGINT,
+    container_health_status VARCHAR(32) NOT NULL DEFAULT "N/A (Not Applicable)",
```

Comment:
should the default be empty string, which just means no health status?

Reply:
if there is no such info, we show not applicable instead of empty

Reply:
it is easier to put the content in the db and show directly on ui.

Reply:
I think it's better to have the default to empty string to save storage, we can show the string in the UI if the value is empty.

Reply:
updated